### PR TITLE
Video step

### PR DESCRIPTION
**Pull request by karenc**
- Create video fragments from a cnx page
    
    In an assessed feature, there is either a video or some text plus an
    exercise.  Add a video fragment class that inherits from the text
    fragment class but with a url method that returns the url of the video
    from .ost-embed. Code assumes only one video in each fragment.

- Add tasked video model and representer

### CODE REVIEW

File: app/representers/api/v1/tasked_video_representer.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module Api::V1
+  class TaskedVideoRepresenter < TaskedReadingRepresenter
+
+    include TaskStepProperties
+
+    property :video_url,
+             type: String,
+             writeable: false,
+             readable: true,
+             as: :video_url,
```
Comment (jpslav) [inline app/representers/api/v1/tasked_video_representer.rb]: I don't think you need this since it is the same name as the `property`?

Comment (karenc) [inline app/representers/api/v1/tasked_video_representer.rb]: Oh i see, thanks, I'll remove it

**Comment by philschatz** [comment]
are these video links `<a class="ost-video" href="...">` or in a `<FOO data-type="media" class="ost-video"><iframe src="..."/></FOO>` (hoping it's the latter ; )

**Comment by karenc** [comment]
It was really just a link to the video, not embedded but we have youtube videos now, so I'm going to change the code.

**Pull request merged**